### PR TITLE
Build/upgrade deps

### PR DESCRIPTION
**Pull request by gregmartDOTin**

### CODE REVIEW

File: project/app/components/MemberInfo.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames';
 import { SpacesLogo } from '@availity/spaces';
 import { useQuery } from 'react-query';
 
-import { useAppContext } from '@/context';
+import { useAppContext } from '../context';
```
Comment (jordan-a-young) [inline project/app/components/MemberInfo.js:8]: I don't think we should change these imports. We can fix it by adding `tsconfig-paths-webpack-plugin` to workflow

**Review by jordan-a-young — CHANGES_REQUESTED** [review]

File: project/app/components/MemberInfo.js (hunk 1 above)
Comment (jordan-a-young) [reply project/app/components/MemberInfo.js:8]: I added this: `plugins: [new TsconfigPathsPlugin()]`, to the `webpack.config.js` file under `config.resolve`.

Comment (gregmartDOTin) [reply project/app/components/MemberInfo.js:8]: Sounds good, I'll update. Thanks

File: package.json [hunk 2]
```diff
@@ -36,41 +36,40 @@
...
   },
   "devDependencies": {
     "@availity/mock-server": "^7.1.4",
-    "@availity/workflow": "^9.0.7",
-    "@testing-library/react": "^12.1.2",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^14.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
+    "@availity/workflow": "^9.5.3-alpha.6",
```
Comment (gregmartDOTin) [inline package.json:65]: I won't merge until I've v10 is cut

File: project/app/components/MemberInfo.js (hunk 1 above)
Comment (gregmartDOTin) [reply project/app/components/MemberInfo.js]: this has been fixed.

**Pull request merged**